web/src/admin/AdminInterface/AdminSidebar.ts

```diff
@@ -112,7 +113,7 @@ export class AkAdminSidebar extends WithCapabilitiesConfig(AKElement) {
 
         // prettier-ignore
         const sidebarContent: SidebarEntry[] = [
-            ["/if/user/", msg("User interface"), { "?isAbsoluteLink": true, "?highlight": true }],
+            [`${globalAK().api.base}if/user/`, msg("User interface"), { "?isAbsoluteLink": true, "?highlight": true }],
```

Comment:
There is also the impersonation trigger where we current redirect to `/`

Reply:
https://github.com/goauthentik/authentik/pull/8675/files#diff-153c71d69a43c3df5e85a6ec763a2ff5e1b7e168ef4975ef1915f13131ad9635 fixed